### ISSUE TITLE
Bug: extracted page title

### ISSUE THREAD

**Issue by dzhuang** [inducer/relate#498]
Extracted page title should
1. be able to be started by characters like `[ ] ! `
1. strip html tags after converted from markdown, which prevent weird html title like `**Important**` and `<span>foo</span>`

**Comment by dzhuang** [comment]
For example, the following markup would fail

```yaml
type: Page
id: welcome

content: |

    # **Important**
    Don't be scared.

```

**Comment by inducer** [comment]
Fail how?

**Comment by dzhuang** [comment]
> Fail how?

`ValidationError`: "no title found in body or title attribute".

**Comment by dzhuang** [comment]
Because the Regex `TITLE_RE = re.compile(r"^\#+\s*(\w.*)", re.UNICODE)` won't match `# **Important**`.

I think `re.compile(r"^\#+\s*(.+)", re.UNICODE)` will do the work.

**Comment by inducer** [comment]
OIC

**Comment by dzhuang** [comment]
Fixed via 87c2ce46a996aaaedaf7bce3a66d530c4f28444d.
